e2e/fragments/AtVideoView.js

```diff
@@ -25,6 +32,22 @@ module.exports = {
     return locator ? locate(locator).inside(this.locateVideoContainer()) : this.locateVideoContainer();
   },
 
+  seekStepForwardSelector() {
+    return locate(this._seekStepForwardSelector).inside(this.locateRootSelector());
+  },
+
+  seekStepBackwardSelector() {
+    return locate(this._seekStepBackwardSelector).inside(this.locateRootSelector());
+  },
+
+  playButtonSelector() {
+    return locate(this._playButtonSelector).inside(this.locateRootSelector());
+  },
```

Comment:
I would say that it's locators instead of selectors. As well that the other ones that use `locate`. Just for consistency. But yeah it's not critical.